plugins/snapd/README.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Plugin: snapd
+---
+
+## Introduction
+
+The plugin provides integration with snapd. 
+
+## Update Behavior
+
+When devices such as UEFI DBX are being updated, the snapd plugin will reach out
+to snapd and pass the update payload, such that the FDE implemented by snapd can
+properly update the TPM policies.
+
+## External Interface Access
+
+This plugin requires:
+
+* access to /run/snapd.socket
+* access to /run/snapd-snap.socket
```

Comment:
is there a specific snapd version required?  If so it should be somewhere in this `README.md`